contracts/utils/DAOTracker.sol

```diff
@@ -0,0 +1,92 @@
+pragma solidity ^0.5.4;
+
+import "@daostack/infra/contracts/Reputation.sol";
+import "../controller/DAOToken.sol";
+import "../controller/Avatar.sol";
+import "../controller/ControllerInterface.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+/**
+ * @title An on-chain "source of truth" for what DAOs
+ *        should be index into DAOstack's subgraph.
+ */
+contract DAOTracker is Ownable {
+
+    struct TrackingInfo {
+        DAOToken            nativeToken;
+        Reputation          nativeReputation;
+        ControllerInterface controller;
+
+        // `blacklist` this from the subgraph's cache
+        bool blacklist;
+    }
+
+    // Mapping from Avatar address to a flag letting the subgraph
```

Comment:
Mapping from Avatar address to TrackingInfo

Reply:
Fixed